language/move-model/tests/testsuite.rs

```diff
@@ -17,13 +21,36 @@ fn test_runner(path: &Path) -> datatest_stable::Result<()> {
         String::from_utf8_lossy(&writer.into_inner()).to_string()
     } else {
         // check that translating from bytecodes also works + yields similar results
-        let modules = env.get_bytecode_modules().cloned().collect();
+        let modules = env.get_bytecode_modules();
         let bytecode_env = run_bytecode_model_builder(modules)?;
         assert_eq!(bytecode_env.get_module_count(), env.get_module_count());
         for m in bytecode_env.get_modules() {
-            let other_m = env.get_module(m.get_id());
+            let raw_module = m.get_verified_module();
+            let other_m = env
+                .find_module_by_id(&raw_module.self_id())
```

Comment:
`find_module_by_id` is not available in envs and can be confusing (as the `ModuleId` in Move model is not the same one as the `ModuleId` in file format...). We may want to expose `GlobalEnv::to_module_name()` and then we can do `env.find_module(env.to_module_name(&raw_module.self_id()))` here.

Reply:
My bad, I had added this function in a different commit + forgot to include it. I will fix that and use a different name to avoid confusion with the Move model's `ModuleId`.